media-gfx/iec16022/iec16022-0.2.7.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="2D barcode/DataMatrix generator"
+DESCRIPTION="With iec16022 you can produce 2D barcodes, also known as \
+DataMatrix, as defined in ISO/IEC 16022."
+HOMEPAGE="https://datenfreihafen.org/projects/iec16022.html"
+COPYRIGHT="2006 Stefan Schmidt"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/rdoeffinger/iec16022/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="34f00da5f6c83dd14b2803e5ed0f5310310e452023fe989c59fc5489b78cd33a"
+SOURCE_FILENAME="iec16022-$portVersion.tar.gz"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	iec16022$secondaryArchSuffix = $portVersion
+	lib:libiec16022$secondaryArchSuffix = 0.1.0 compat >= 0
+	cmd:iec16022 = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libpopt$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	iec16022${secondaryArchSuffix}_devel
+	devel:libiec16022$secondaryArchSuffix = 0.1.0 compat >=0
```

Comment:
Good catch!

@xdizzaster, before you push the fix with the space, if you would like to replace
```
	libtoolize --force --copy --install
	aclocal
```
by
```
	aclocal
	libtoolize --force
```
and
```
	autoconf
	automake --add-missing
```
by
```
	automake --add-missing
	autoconf
```
please do not hesitate.
I know it won't make much difference. The resulting packages are even identical.
It's just that it will be closer to what the author suggests in autogen.sh.
After all, he is the author of a great utility, and I feel a little bit guilty of having asked you to put the block in a different order than that recommended by the author.

@korli, do you think these cosmetic changes are welcome?

Thanks!